packages/web-react/src/components/Tooltip/demo/TooltipIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
```

Comment:
can you unify the usage of useState?
Sometimes it is used like here and sometimes as `React.useState`.

Reply:
In this component, we can create an issue to do it globally.

Reply:
fixed